app/controllers/concerns/course/lesson_plan/personalization_concern.rb

```diff
@@ -71,11 +91,13 @@ def algorithm_fomo(course_user)
             personal_point + (reference_time.start_at - reference_point) * learning_rate_ema,
             course_tz,
             FOMO_DATE_ROUNDING_THRESHOLD
-          )
+          ) if personal_time.start_at > Time.zone.now
```

Comment:
Should this be based on the user's timezone or the course timezone instead?

For an example of course timezone use, see https://github.com/Coursemology/coursemology2/blob/master/app/jobs/consolidated_item_email_job.rb#L10

Reply:
Same for the other `Time.zone.now` comparisons below.

Reply:
This should be comparison between two datetimes with zone, so any difference in timezone should be handled by rails.